src/Command/Migration/DownCommand.php

```diff
@@ -18,21 +18,16 @@
 final class DownCommand extends BaseMigrationCommand
 {
     protected static $defaultName = 'migrate/down';
+    protected static $defaultDescription = 'Rolls back the last applied migration';
 
-    private EventDispatcherInterface $eventDispatcher;
+    private ?EventDispatcherInterface $eventDispatcher;
 
-    public function __construct(CycleDependencyProxy $promise, EventDispatcherInterface $eventDispatcher)
+    public function __construct(CycleDependencyProxy $promise, ?EventDispatcherInterface $eventDispatcher = null)
```

Comment:
Could you tell me why eventDispatcher must be optional?

Reply:
Reverted.